### DISCUSSION TITLE
5월 31일 회의할 내용

### DISCUSSION THREAD

**Discussion by gracefulBrown** [woowacourse/prolog#71] [Notice]
## 이슈 처리 진행 상황 공유
- 에디터 이미지 적용 진행 상황 https://github.com/woowacourse/prolog/issues/62
- 로그인 callback 처리를 비롯한 로그인 정상 동작 여부 https://github.com/woowacourse/prolog/issues/59
- 포스트 작성 화면 조회 시 카테고리, 태그 목록 노출 기능 https://github.com/woowacourse/prolog/issues/60
- 포스트 작성 요청 시 모든 정보(카테고리, 태그, 사용자 등)가 포함되어 저장되는지 여부 https://github.com/woowacourse/prolog/issues/65, https://github.com/woowacourse/prolog/issues/60
- 카테고리, 태그 별 필터 기능 https://github.com/woowacourse/prolog/issues/61
- 배포 방법 문서화 https://github.com/woowacourse/prolog/issues/70

## 커뮤니케이션 방식에 대한 회고
- 프론트엔드와 백엔드 간의 협업 시 소통에 대해서 회고
- 개선 포인트가 있다면 더 나은 소통 방안에 대한 논의
  - ex) 개발 및 협업 프로세스, 회의 주기나 방법, 진행 상황 공유 방법 등

**Comment by sunhpark42** [comment]
- 에디터 이미지 적용 진행 상황 #62
    - 진행 x
* 로그인 callback 처리를 비롯한 로그인 정상 동작 여부 #59
    - 시간이 오래 걸릴 것 같아서 넘김
* 포스트 작성 화면 조회 시 카테고리, 태그 목록 노출 기능 #60
    - API : 로컬주소/docs/index.html 로 접속후 확인
    - ["string". "string"... ] => [{id: 0, name: 'string'}...]
* 포스트 작성 요청 시 모든 정보(카테고리, 태그, 사용자 등)가 포함되어 저장되는지 여부 #65, #60
    - 로그인이 완성 되어야 오고가고가 설정될 것 같음.
    - 이전과 유사한 형태로 진행 될 듯.
* 카테고리, 태그 별 필터 기능 #61
```
filter : {
    tag: [{id: 0, name: string}, ...],
    mission: [{id: 0, name : string}, ...],
    ...
}
```
* 배포 방법 문서화 #70
    - 브라운이 정리해주심

* category 를 보다 의미에 맞게 변경 => topic, mission, project, category, subject => mission
* 카테고리 이름 => Frontend Lv1 자동차 경주 / Backend Lv1 체스게임 ? => 우선은 나누지 않고, 후에 불편하면 나누기.
* 대주제 => 소주제 로 정하는 것도 좋을 것 같다.
